client/src/sections/courses/ContentSection.tsx

```diff
@@ -1,92 +1,42 @@
+import React, { Fragment } from 'react';
 import Iframe from '@/src/shared/components/Iframe';
 import SidebarContent from '@/src/shared/components/SidebarContent';
 import SideBar from '@/src/shared/components/SidebarContent/SideBar';
-import React, { Fragment } from 'react';
+import { useGetLessonByCourseIDQuery } from '@/services/lessonApi';
+import { useRouter } from 'next/router';
+import LoadingSpinner from '@/src/shared/components/LoadingSpinner';
 
 const ContentSection: React.FC = () => {
+  const router = useRouter();
+  const params = router.query;
+
+  const { data: lesson, isLoading } = useGetLessonByCourseIDQuery(params.id);
```

Comment:
```suggestion
  const { data: lesson, isLoading } = useGetLessonByCourseIDQuery(params.id, {skip: params.id === undefined});
```

Please avoid sending api request when id is still undefined

![image](https://github.com/framgia/sph-lms/assets/110364690/fa167d5d-1696-4e2e-85fc-1022dcc92ae7)